client/src/test/java/com/influxdb/client/internal/InfluxQLQueryApiImplTest.java

```diff
@@ -70,12 +74,19 @@ void readInfluxQLResult() throws IOException {
 				"name,tags,time,usage_user,usage_system\n" +
 				"cpu,\"region=us-east-1,host=server1\",1483225200,13.57,1.4\n" +
 				"cpu,\"region=us-east-1,host=server1\",1483225201,14.06,1.7\n" +
-				"cpu,\"region=us-east-1,host=server2\",1483225200,67.91,1.3\n");
+				"cpu,\"region=us-east-1,host=server2\",1483225200,67.91,1.3\n" +
+			  "\n" +
+			  "name,tags,key\n" + //emulate SHOW SERIES response
+			  ",,temperature\n" +
+			  ",,\"pressure\"\n" +
+			  ",,humid\n" +
+			  ",,\"temperature,locale=nw002,device=rpi5_88e1\""
+			);
```

Comment:
Please add this as new test, something like `testShowSeries`. It will be better for future maintenance of `InfluxQLQueryApiImplTest.java`.